tests/HubSession.spec.ts

```diff
@@ -53,13 +53,18 @@ describe('HubSession', () => {
         resolver: mockResolver
       });
       
+      const kid = 'testkey';
+      const keyStore: IKeyStore = new KeyStoreMem();
+      keyStore.save(kid, clientPrivateKey);
```

Comment:
not related. The wallet developer can come up with its own scheme for this. DID's would be insufficient because a DID can have e.g. a signing and an encryption key and both keys need to be referenced.